.github/workflows/main/installDependencies.py

```diff
@@ -63,6 +61,13 @@
 	default = defaultURL,
 )
 
+parser.add_argument(
+	"--variant",
```

Comment:
It's a bit weird how `--variant` only works if you don't use `--archiveURL`. Maybe we should replace `--archiveURL` with `--version` (and possibly add a `--repo` argument defaulting to `GafferHQ/dependencies`)?